### PR TITLE
Fixes #145 Add batch files for Single/Multiple Oral with Table dissolution

### LINKED ISSUE #145: Add batch files for Single/Multiple Oral with Table dissolution

### LINKED ISSUE #146: Add batch files for Single/Multiple Oral with solubility table

### LINKED ISSUE #147: Add batch files for AGP Ontogeny

### PR DESCRIPTION
**Pull request by Yuri05**
Fixes #146 Add batch files for Single/Multiple Oral with solubility table
Fixes #147 Add batch files for AGP Ontogeny

### CODE REVIEW

**Comment by Yuri05** [comment]
Having new snapshot functionality makes batch tests creation so much easier :smile: :1st_place_medal:

File: data/Inputs/BatchFiles/Human_pH_SolubilityTable.json [hunk 1]
```diff
@@ -0,0 +1,650 @@
...
+          "Unit": "year(s)"
+        }
+      },
+      "Parameters": [
+        {
+          "Path": "Organism|Lumen|Caecum|pH",
+          "Value": 5.0,
+          "ValueOrigin": {
+            "Id": 28,
+            "Source": "Unknown"
```
Comment (msevestre) [inline data/Inputs/BatchFiles/Human_pH_SolubilityTable.json]: How can we have a source unknown coming from DB??

Comment (Yuri05) [reply data/Inputs/BatchFiles/Human_pH_SolubilityTable.json]: Value was overwritten by user, so the Source changed from Publication to Unknown.

Comment (msevestre) [reply data/Inputs/BatchFiles/Human_pH_SolubilityTable.json]: ok makes sense

**Pull request merged**